packages/api/database/migrations/1605981580414_chat_message.js

```diff
@@ -0,0 +1,26 @@
+/** @type {import('@adonisjs/lucid/src/Schema')} */
+const Schema = use('Schema');
+
+class ChatMessages extends Schema {
+	up() {
+		this.create('chat_messages', (table) => {
+			table.increments();
+			table
+				.uuid('chatId')
+				.references('id')
+				.inTable('chats')
+				.onUpdate('cascade')
+				.onDelete('cascade');
+			table.string('type');
+			table.integer('fromUserId');
```

Comment:
To match the rest of the project
```suggestion
			table
				.uuid('chat_id')
				.references('id')
				.inTable('chats')
				.onUpdate('cascade')
				.onDelete('cascade');
			table.string('type');
			table.integer('from_user_id');
```